### DISCUSSION TITLE
Change Number

### DISCUSSION THREAD

**Discussion by raminhos** [project-error/npwd#589] [General] [answered]
I have a snippet to use a sim card and add/change number to database.
How can we refresh phone so he assume new number instead of player restart fivem ?

**Comment by TasoOneAsia** [comment, marked as answer]
This would require some changes to the sim card system, but essentially all that is required is dispatching an event that updates phone number accordingly. But, there may be concerns with other items that rely on `phone_number` for identification.

**Comment by raminhos** [comment]
Yes, i tried the unload and load player export but didn't work. Player must restart game so the phone can read the new number

**Reply by TasoOneAsia** [reply]
Unload function isn't functioning, see the issue #476
